airbyte-integrations/connectors/source-bigcommerce/source_bigcommerce/schemas/products.json

```
@@ -0,0 +1,1395 @@
+{
+    "type": "object",
+    "title": "Product Response",
+    "properties": {
```

```suggestion
    "required": ["name", ...]
    "properties": {
```
You should list required fields this way, do not use x-required please.

---

done